CHANGELOG.md

```diff
@@ -0,0 +1,179 @@
+# Changelog
+
+## v1.7.5
+
+- ✨add an option to silent multiple Init errors (#310)
```

Comment:
it could be nice to be able add the link the PR number, it does not seem to be generated from github, cf [CHANGELOG](https://github.com/DataDog/browser-sdk/blob/b7600a0f4169bb04d23c5238752bd07b23841747/CHANGELOG.md#v175)

Reply:
Github should be able to recognize them once it's on master. 
I don't think their PR parsing engine is running on every document on every branches.

Reply:
It does not, I found this one https://github.com/angular/protractor/blob/master/CHANGELOG.md#features

Reply:
I've added a pull request links.